pages/api/polling/vote.ts

```diff
@@ -45,68 +64,68 @@ export default withApiHandler(
       const { voter, pollIds, optionIds, nonce, expiry, signature, network, secret } = req.body;
 
       if (typeof voter !== 'string' || !voter) {
-        const error = { message: { error: API_VOTE_ERRORS.VOTER_MUST_BE_STRING } };
-        postError(JSON.stringify({ ...error.message, ...req.body }));
+        const error = { error: API_VOTE_ERRORS.VOTER_MUST_BE_STRING };
```

Comment:
I think the error object typically has a message property instead of `error`. Most examples I've seen look more like this:
```
error: {
    code: 1150,
    message: 'bla bla bla'
  }
}
```